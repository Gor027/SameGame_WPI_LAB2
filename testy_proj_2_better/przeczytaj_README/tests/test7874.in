  9A AA9A  99 9 99 999A 9 AA9 99 9999A  A 99AAA AA
99A9A 99A999 A9
