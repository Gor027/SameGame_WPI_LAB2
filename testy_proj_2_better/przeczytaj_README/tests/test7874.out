9A  A9A  9  9
A9AA99A9999A999999A9AA9999999AA99AAAAA
